packages/katello/katello/hostname-change.rb

```diff
@@ -167,6 +167,28 @@ def using_custom_certs?(scenario_answers)
       scenario_answers["certs"]["server_cert_req"]
     end
 
+    def delete_puppet_certs
+      puppet_ssldir = @scenario_answers['puppet']['ssldir']
+      files = [
+        'ca/ca_crt.pem',
+        'ca/ca_crl.pem',
+        'ca/infra_crl.pem',
+        'certs/ca.pem',
+        'crl.pem',
+        'ca/ca_pub.pem',
+        'ca/inventory.txt',
+        'ca/infra_inventory.txt',
+        'ca/infra_serials',
```

Comment:
I don't see this on my Puppet 5 setup and I think you're using Puppet 6. Do we want to be safe and just remove the entire `puppet_ssldir`? AFAIK Puppet recreates it if needed and you don't need to track individual files.